backend/app/assets/stylesheets/spree/backend/components/_icons.scss

```diff
@@ -1,257 +1,219 @@
-// TODO: Switch everything to use glyphicons
-.icon, .glyphicon {
-  vertical-align: text-top;
+
+// Spree Icon Default
+.icon {
+  display: inline-block;
+  width: 1rem;
+  height: 1rem;
+  overflow: hidden !important;
+  vertical-align: middle !important;
+  background-repeat: no-repeat !important;
+  background-size: 100% !important;
+  background-position: center !important;
 }
 
-// Maps Glyphicons to Spree .icon prefix.
-// You can apply same pattern to extend with more icons.
-//
-// example:
-// glyphicon glyphicon-pencil => icon icon-edit
-//
+// Spree Icon Sizing
+.icon-xs {
+  width: 0.6rem !important;
+  height: 0.6rem !important;
+}
 
-// All Glyphicons Halflings in Bootstrap v3.3.1.0
-$default_icons: (
-  adjust
-  align-center
-  align-justify
-  align-left
-  align-right
-  arrow-down
-  arrow-left
-  arrow-right
-  arrow-up
-  asterisk
-  backward
-  ban-circle
-  barcode
-  bell
-  bold
-  book
-  bookmark
-  briefcase
-  bullhorn
-  calendar
-  camera
-  certificate
-  check
-  chevron-down
-  chevron-left
-  chevron-right
-  chevron-up
-  circle-arrow-down
-  circle-arrow-left
-  circle-arrow-right
-  circle-arrow-up
-  cloud
-  cloud-download
-  cloud-upload
-  cog
-  collapse-down
-  collapse-up
-  comment
-  compressed
-  copyright-mark
-  credit-card
-  cutlery
-  dashboard
-  download
-  download-alt
-  earphone
-  eject
-  envelope
-  euro
-  exclamation-sign
-  expand
-  export
-  eye-close
-  eye-open
```

Comment:
I found 2 occurrences of this icon in https://github.com/spree/spree/blob/master/backend/app/helpers/spree/admin/base_helper.rb. `icon: 'view'` seems ok.